packages/stream-api/src/webSocketClient.ts

```diff
@@ -15,12 +15,18 @@ export class WebSocketClient {
   private webSocket?: WebSocket;
   private onMessageDispatcher = new EventDispatcher<MessageEvent>();
   private reopenAttempt = 0;
+  private urlProvider?: UrlProvider;
+  private timer?: NodeJS.Timer;
```

Comment:
This type isn't correct because we're not in a Node environment. I would recommend changing the type to `number` and using `window.setTimeout`.

Reply:
![image](https://user-images.githubusercontent.com/49169871/89433306-c9497980-d707-11ea-8759-d119949ced1e.png)

this is the type provided by `setTimeout`

Reply:
used window.setTimeout resolves this. addressed.